sdk/testutil/testlog.go

```diff
@@ -7,7 +7,7 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/hashicorp/go-hclog"
+	hclog "github.com/hashicorp/go-hclog"
```

Comment:
why ? 🙈

Reply:
I thought this was causing a linting error since the package is used as `hclog` but looks like I was wrong. Great catch!